<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>581</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearPushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolation sort</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time (millis):</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shufflePushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>260</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolation search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>300</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Index:</string>
   </property>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>330</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>380</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>BinPow</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Result:</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>480</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
